src/WebJobs.Script/Binding/CoreExtensionsScriptBindingProvider.cs

```diff
@@ -66,6 +66,7 @@ public override Collection<Attribute> GetAttributes()
 
                 string schedule = Context.GetMetadataValue<string>("schedule");
                 bool runOnStartup = Context.GetMetadataValue<bool>("runOnStartup");
+                bool useMonitor = Context.GetMetadataValue<bool>("useMonitor", true);
```

Comment:
I went to go do that, then I realized that page is woefully out of date, and isn't really set up like host.json to be a documentation file. I've logged https://github.com/Azure/azure-webjobs-sdk-script/issues/1324